src/Core/src/Platform/Windows/Styles/Resources.xaml

```diff
@@ -11,14 +11,14 @@
         <ResourceDictionary Source="MauiComboBoxStyle.xaml" />
         <ResourceDictionary Source="MauiSliderStyle.xaml" />
         <ResourceDictionary Source="MauiStepperStyle.xaml" />
+        <ResourceDictionary Source="NavigationRootViewStyle.xaml" />
     </ResourceDictionary.MergedDictionaries>
 
     <x:Boolean x:Key="MicrosoftMauiCoreIncluded">true</x:Boolean>
 
     <uwp:ColorConverter x:Key="ColorConverter" />
     
-    <Style x:Key="MauiRootContainerStyle" TargetType="Panel">
-		<Setter Property="Background" Value="{ThemeResource ApplicationPageBackgroundThemeBrush}" />
+    <Style x:Key="MauiRootContainerStyle" TargetType="Grid">
```

Comment:
Once Mica is available we'll use that for our background styles
https://docs.microsoft.com/en-us/windows/apps/design/style/mica

I had to remove the Background property here because it'll cause the min/max/close buttons to become unclickable